CONTRIBUTING.md

```diff
@@ -124,3 +124,73 @@ what a commit is about.
 
 When you create a pull request, please
 [make your PR editable](https://github.com/blog/2247-improving-collaboration-with-forks).
+
+## Rebasing
+
+If other PRs have been merged during the time between your initial PR creation
+and final approval, it may be required that you rebase your changes against the
+latest `main` branch.
```

Comment:
Maybe explain when exactly a rebase is required.

Reply:
Yes, only if merge conflicts have arisen.